lib/seahorn/BvOpSem2RawMemMgr.hh

```diff
@@ -12,7 +12,10 @@
 namespace seahorn {
 namespace details {
 
-class RawMemManager : public OpSemMemManager {
+class OpSemMemRepr;
+class OpSemAllocator;
+
+class RawMemManagerCore : public CoreMemManager {
```

Comment:
is it `CoreMemManager` or `MemManagerCore` ? Would be nice to be consistent with `RawMemManagerCore` if the idea is that `Core` is the base of everything

Reply:
fixed.